chapters/annotations.tex

```diff
@@ -546,21 +571,23 @@ \subsection{Single Use of Class}\label{annotation-for-single-use-of-class}\label
 For state machines it is useful to have single instances of local classes.
 This can be done using:
 \begin{lstlisting}[language=modelica]
-annotation(singleInstance = true)
+/*literal*/ constant Boolean singleInstance = false;
 \end{lstlisting}
 
-The annotation \fmtannotationindex{singleInstance} in a class indicates that there should only be one component instance of the class, and it should be in the same scope as the class is defined.
+The annotation \fmtannotationindex{singleInstance}, if \lstinline!true!, in a class indicates that there should only be one component instance of the class, and it should be in the same scope as the class is defined.
 The intent is to remove the class when the component is removed and to prevent duplication of the component.
+If \lstinline!false!, it has no impact.
 
 \subsection{Connection Restrictions}\label{connection-restrictions}
 
 A connector component declaration may have the following annotation:
 \begin{lstlisting}[language=modelica]
-annotation(mustBeConnected = "message");
+/*literal*/ constant String mustBeConnected;
 \end{lstlisting}%
 \annotationindex{mustBeConnected}
 
 It makes it an error if the connector does not appear as an inside connector in any connect-equation (for a conditional connector this check is only active if the connector is enabled).
```

Comment:
I feel we are ending up with a cumbersome notation if we always need to say that the annotation only has effect when present…
```suggestion
Only has effect when present, and makes it an error if the connector does not appear as an inside connector in any connect-equation (for a conditional connector this check is only active if the connector is enabled).
```

Reply:
I added a default and rephrased.